### PR TITLE
LOK-2624: Create static MonitoredEntityProvider repository and provider

### PR DESCRIPTION
**Pull request by fooker**
## Jira link(s)
- https://opennms.atlassian.net/browse/LOK-2624

## Checklist
* [ ] Follows Lōkahi's [development guidelines.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS-Cloud/lokahi/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts.
* [ ] Notify documentation team of any changes to names of screens or features (affects URLs).

### CODE REVIEW

File: inventory/main/src/main/resources/db/changelog/hs-0.1.0/tables/simple-monitored-entity.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                <constraints nullable="false"/>
+            </column>
+
+            <column name="location_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+
+            <column name="type" type="text" />
+
+            <column name="config" type="clob" />
```
Comment (cgorantla) [inline inventory/main/src/main/resources/db/changelog/hs-0.1.0/tables/simple-monitored-entity.xml:25]: type can be jsonb

File: inventory/main/src/main/resources/db/changelog/hs-0.1.0/tables/simple-monitored-entity.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+    <changeSet author="fooker" id="0.1.2-create-simple-monitored-entity">
+        <createTable tableName="simple_monitored_entity">
+            <column name="id" type="UUID" defaultValueComputed="gen_random_uuid()">
```
Comment (cgorantla) [inline inventory/main/src/main/resources/db/changelog/hs-0.1.0/tables/simple-monitored-entity.xml:9]: currently all primary keys are bigint , any particular reason for this to be UUID ?

File: inventory/main/src/main/java/org/opennms/horizon/inventory/monitoring/simple/SimpleMonitoredEntity.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+public class SimpleMonitoredEntity {
+    @Id
+    private UUID id;
+
+    private String name;
+
+    private String tenantId;
+    private long locationId;
+
+    private String type;
```
Comment (cgorantla) [inline inventory/main/src/main/java/org/opennms/horizon/inventory/monitoring/simple/SimpleMonitoredEntity.java:42]: Can this be an Enum ?

**Review by cgorantla — COMMENTED** [review]
few comments otherwise LGTM!

File: inventory/main/src/main/resources/db/changelog/hs-0.1.0/tables/simple-monitored-entity.xml (hunk 2 above)
Comment (fooker) [reply inventory/main/src/main/resources/db/changelog/hs-0.1.0/tables/simple-monitored-entity.xml:9]: Mixture between POC and making graphql endpoints easier to use.

**Pull request merged**